### PR TITLE
ProtoMsg moves in.

### PR DESCRIPTION
**Pull request by merlin-northern**
goes with:
 * gui: https://github.com/mendersoftware/gui/pull/1110
 * mender-shell: https://github.com/mendersoftware/mender-shell/pull/18

ChangeLog:none
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ const (
 	// SettingListen is the config key for the listen address
 	SettingListen = "listen"
 	// SettingListenDefault is the default value for the listen address
-	SettingListenDefault = ":8080"
+	SettingListenDefault = ":11080"
```
Comment (alfrunes) [inline config/config.go:25]: Was this committed by mistake?

File: config/config.go [hunk 2]
```diff
@@ -32,7 +32,7 @@ const (
 	// SettingMongo is the config key for the mongo URL
 	SettingMongo = "mongo_url"
 	// SettingMongoDefault is the default value for the mongo URL
-	SettingMongoDefault = "mongodb://mender-mongo:27017"
+	SettingMongoDefault = "mongodb://127.0.0.1:27017"
```
Comment (alfrunes) [inline config/config.go:35]: Here also.

File: config/config.go (hunk 1 above)
Comment (merlin-northern) [reply config/config.go:25]: oh crap, yes, I committed my local changes. sorry about that. good catch.

File: config/config.go (hunk 2 above)
Comment (merlin-northern) [reply config/config.go:35]: same thing. thanks and sorry.

File: api/http/device.go [hunk 3]
```diff
@@ -135,7 +136,7 @@ func (h DeviceController) Connect(c *gin.Context) {
 	}
```
Comment (alfrunes) [inline api/http/device.go:136]: Can you change the `Upgrader.Subprotocols` from `[]string{"binary"}` to `[]string{"protomsg/msgpack"}`?

File: api/http/management.go [hunk 4]
```diff
@@ -154,7 +155,7 @@ func (h ManagementController) ConnectDevice(
 			rest.RenderError(c, s, e)
 		},
 	}
```
Comment (alfrunes) [inline api/http/management.go:157]: Can you change the `Upgrader.Subprotocols` from `[]string{"binary"}` to `[]string{"protomsg/msgpack"}`?

**Review by alfrunes — APPROVED** [review]
LGTM.
I just have one small request (pre-approving).

File: api/http/device.go (hunk 3 above)
Comment (merlin-northern) [reply api/http/device.go:136]: [sure](https://github.com/mendersoftware/deviceconnect/pull/13/commits/03dd6c7c927d58219cb608e8f1f708c78bd11362) man

File: api/http/management.go (hunk 4 above)
Comment (merlin-northern) [reply api/http/management.go:157]: [sure](https://github.com/mendersoftware/deviceconnect/pull/13/commits/03dd6c7c927d58219cb608e8f1f708c78bd11362) man

**Pull request merged**